From 9106e17b62e2881daa290f1bd7e0c63eb25293be Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan@agner.ch>
Date: Mon, 29 Apr 2024 17:32:02 +0200
Subject: [PATCH] Allow to build chip-cert with boringssl

Currently when trying to build chip-cert with boringssl gn doesn't
generate a ninja target. This is since `chip_can_build_cert_tool` gets
set to false.

Allow to build with boringssl just like with openssl.
---
 build/chip/tools.gni | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/chip/tools.gni b/build/chip/tools.gni
index efb25d3fa4..46c321d965 100644
--- a/build/chip/tools.gni
+++ b/build/chip/tools.gni
@@ -19,7 +19,7 @@ declare_args() {
   chip_build_tools = current_os != "freertos" && current_os != "android" &&
                      chip_device_platform != "fake"
   chip_can_build_cert_tool =
-      chip_crypto == "openssl" ||
+      chip_crypto == "openssl" || chip_crypto == "boringssl" ||
       (chip_crypto == "" &&
        (current_os != "android" && current_os != "freertos" &&
         current_os != "zephyr" && current_os != "mbed" &&
-- 
2.44.0

